draft-irtf-cfrg-vdaf.md

```diff
@@ -3062,6 +3062,129 @@ measurement is sharded. This is provided to the FLP by Prio3.
 | `Field`          | `Field128` ({{fields}}) |
 {: title="Parameters of validity circuit Histogram."}
 
+### Prio3SumVec
+
+This instance of Prio3 supports summing a vector of integers. It has three
+parameters, `length`, `bits`, and `chunk_length`. Each measurement is a vector
+with length equal to the `length` parameter. Each element of the measurement is
+an integer in the range `[0, 2^bits)`. It is RECOMMENDED to set `chunk_length`
+to an integer near to the square root of `length * bits`. The optimal choice for
+any measurement size will vary slightly due to rounding.
+
+This instance uses PrgSha3 ({{prg-sha3}}) as its PRG. Its validity circuit,
+denoted `SumVec`, uses `Field128` ({{fields}}) as its finite field.
+
+Measurements are encoded as a vector of field elements with length
+`length * bits`. The field elements in the encoded vector represent all the bits
+of the measurement vector's elements, consecutively, in LSB to MSB order:
+
+~~~
+def encode(SumVec, measurement: Vec[Unsigned]):
+    if len(measurement) != SumVec.length:
+        raise ERR_INPUT
```

Comment:
Orthogonal to this PR, but: WDYTA about using normal exceptions for these kinds of things?
```suggestion
        raise Exception('input: invalid length')
```

Reply:
https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/277